test/e2e/storage/testsuites/multivolume.go

```diff
@@ -157,7 +157,7 @@ func (t *multiVolumeTestSuite) DefineTests(driver TestDriver, pattern testpatter
 	//      [   node1   ]                           ==>        [   node2   ]
 	//          /    \      <- same volume mode                   /    \
 	//   [volume1]  [volume2]                              [volume1]  [volume2]
-	ginkgo.It("should access to two volumes with the same volume mode and retain data across pod recreation on different node", func() {
+	ginkgo.It("should access to two volumes with the same volume mode and retain data across pod recreation on different node [LinuxOnly]", func() {
```

Comment:
In general, we are inconsistent with how we skip tests based on OS. Sometimes, we check node OS, and sometimes we use [LinuxOnly] tag. Can you clarify when do we use which method?

Reply:
If the tests can only work on linux, we mark it as [linuxonly]

for some tests, the test can work on Windows but with certain fstypes (not all fstypes), there is a check here https://github.com/kubernetes/kubernetes/blob/02528ce91af8398999eb9a99b7abff314e4d8b83/test/e2e/storage/testsuites/base.go#L180